types/mini-css-extract-plugin/index.d.ts

```diff
@@ -4,9 +4,11 @@
 //                 Katsuya Hino <https://github.com/dobogo>
 //                 Spencer Miskoviak <https://github.com/skovy>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
+//                 James Garbutt <https://github.com/43081j>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.7
 
+/// <reference types="node" />
```

Comment:
Why you're adding this into the package? I thik there is no diret dependency to Node types here.

Reply:
@sandersn added it, see his second point. not entirely sure what problem it solved though, maybe he can explain

Reply:
webpack@5 depends on node types, but doesn't reference them. I have found 3 ways to fix this:

1. Add the reference here.
2. Add the reference in webpack.
3. Have each user of mini-css-extract-plugin install `@types/node` and add `types: ["node"]` to their tsconfig.

(3) is bad for usability of this package. (1) is .. mysterious. (2) is probably the right fix. I'll look at webpack's setup and see whether it makes sense for it to add the reference.

Even if (2) is right, then it'll have to wait for a new version of webpack. So you could take this change in order to get this PR merged.